server/lib/plans.ts

```diff
@@ -95,6 +95,19 @@ export async function handleHostPlanBankTransfersLimit(host, { throwException =
   }
 }
 
+export async function handleTransferwisePaymentLimit(host, expense, { throwException = false }): Promise<void> {
```

Comment:
**[:ice_cream: Details & personal preferences]** It's a strange API, because if `throwException` is false (default) then the function will never do anything. Should probably either return a boolean in such cases or remove this param.

Reply:
Not a justification but an explanation, The API is copied from `handleHostPlanAddedFundsLimit` which has ability to throwException AND notifyAdmins.

Reply:
Exact, `handleHostPlanAddedFunsLimit` was used as a boilerplate. I thought the consistency was better here but I agree it is misleading.
I'll refactor both functions to remove unused parameters.